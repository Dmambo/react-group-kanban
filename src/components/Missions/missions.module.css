.active {
  color: red;
}

.table {
  margin: 2rem 0;
  border-collapse: collapse;
  width: 100%;
  font-size: 15px;
}

td,
th {
  border: 1px solid #ddd;
  text-align: left;
  padding: 5px;
}

th,
.missionName {
  font-weight: bold;
}

tr:nth-child(even) {
  background-color: #f5f4f4;
}

.missionReserved {
  text-align: center;
  font-size: 14px;
  word-spacing: -2px;
  color: #fff;
  font-weight: bold;
  background-color: #1599ba;
  padding: 5px;
  border-radius: 10px;
}

.notReserved {
  padding: 5px;
  text-align: center;
  font-size: 15px;
  word-spacing: -2px;
  color: #fff;
  font-weight: bold;
  background-color: #383838;
  border-radius: 10px;
}

.leave {
  border: 2px solid rgb(238, 68, 68);
  background-color: transparent;
  color: rgb(238, 68, 68);
  font-size: 15px;
  padding: 5px;
  font-weight: bold;
  border-radius: 5px;
}

.join {
  border: 2px solid rgb(56, 56, 56);
  background-color: transparent;
  color: rgb(56, 56, 56);
  font-size: 15px;
  padding: 5px;
  font-weight: bold;
  border-radius: 5px;
}

.badge {
  width: 10%;
}
