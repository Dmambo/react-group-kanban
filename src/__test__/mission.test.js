import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import store from '../redux/configureStore';
import Missions from '../components/Missions/Missions';

describe('Test Missions Component', () => {
  it('should render Missions component', () => {
    const tree = renderer
      .create(
        <Provider store={store}>
          <Missions />
        </Provider>,
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
