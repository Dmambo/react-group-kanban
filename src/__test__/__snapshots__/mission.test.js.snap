// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Missions Component should render Missions component 1`] = `
<table
  className="table"
>
  <tbody>
    <tr>
      <th>
        Mission
      </th>
      <th>
        Description
      </th>
      <th>
        Status
      </th>
      <th>
         
      </th>
    </tr>
  </tbody>
</table>
`;
